#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_g++.dpatch by  <danchev@spnet.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Use system's g++, versions if any should specified in debian/control

@DPATCH@
diff -urNad bobcat-1.10.0~/INSTALL.cf bobcat-1.10.0/INSTALL.cf
--- bobcat-1.10.0~/INSTALL.cf	2006-07-05 20:47:00.000000000 +0300
+++ bobcat-1.10.0/INSTALL.cf	2006-07-27 23:20:51.000000000 +0300
@@ -51,4 +51,4 @@
     # `COMPILER') the 4.1.x version of the compiler is used (of course,
     # it must have been installed in your system as well). If you want to use
     # your `standard' C++ compiler, simply remove the next line.
-COMPILER=g++-4.1
+#COMPILER=g++-4.1
